plugins/modules/remote_management/ism/is_add_user.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python
+# -*- coding:utf-8 -*-
+
+# Copyright(C) 2020 Inspur Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: is_add_user
+version_added: "2.9"
```

Comment:
This should be a collection version. The next release of community.general is 1.0.0.
```suggestion
version_added: "1.0.0"
```
For your own collection, you have to decide yourself what's the first released version is. I suggest starting with 0.1.0, and release 1.0.0 after some time when you are sure that no breaking changes will happen for some time. (Collection version numbers follow semantic versioning: https://semver.org/)